docs/reference/supported-languages.njk

```diff
@@ -122,7 +123,8 @@ When you scan a codebase, Bearer will automatically select the appropriate langu
           {% endif %}
         </td>
         <td>
-          {{ data.status }}
+          {{ data.status }}<br/>
+          <small><i>{{ data.comment }}</i> </small>
```

Comment:
```suggestion
          <span class="italic text-sm">{{ data.comment }}</span>
```
Try and stick to using tailwind stuff as the browser styles from these tags are not consistant:
https://tailwindcss.com/docs/font-size

if text-sm is too big you can use `text-[12px]` class for one offs

Reply:
Used xs and looking great, thanks!